app/controllers/chores_controller.rb

```diff
@@ -17,6 +19,15 @@ def create
     redirect_to @chore
   end
 
+  def update
+    @chore = Chore.find(params[:id])
+    @chore.lastdone = Date.today
+    @chore.nextdue = @chore.lastdone + @chore.interval_days
+    @chore.updated_at = Time.now
```

Comment:
`.save` doesn't touch the database if none of the values have changed. Since I want to use `updated_at` as a secondary sort value, I need it to change when the user clicks, even if the other two values remain the same.